site/docs/guide/errors.md

```diff
@@ -22,14 +22,14 @@ A more advanced error handling mechanism can be found [down here](#error-boundar
 
 ## Catching Errors
 
-It depends on your setup how to catch errors.
+How you catch errors will depend on your setup.
```

Comment:
It should be only this sentence.

Reply:
Updated in 40ba06f66a9748eb02a2256b01350a62eef0b18a